stable/yugaware/templates/kube-state-metrics-service-account.yaml

```diff
@@ -0,0 +1,14 @@
+{{- if and .Values.kubeStateMetric.install (not .Values.kubeStateMetric.customEndpoint) -}}
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  labels:
+    app: {{ .Release.Name }}-yugaware-ksm
+    chart: {{ template "yugaware.chart" . }}
+    release: {{ .Release.Name }}
+  name: {{ .Release.Name }}-ksm
+{{- if .Values.yugaware.serviceAccountAnnotations }}
+  annotations:
+{{ toYaml .Values.yugaware.serviceAccountAnnotations | indent 4 }}
+{{- end }}
```

Comment:
Here as well. These are annotations for yugaware's service account, let's not reuse those. We can add something like `.Values.kubeStateMetrics.serviceAccountAnnotations` later if there is a need.